src/views/CHConfigEditor.tsx

```diff
@@ -315,6 +327,60 @@ export const ConfigEditor: React.FC<Props> = (props) => {
           </div>
         </div>
       </div>
+      <div className="gf-form-group">
+        <h3>Custom Settings</h3>
+        <br />
+        {
+          customSettings.map(({ setting, value }, i) => {
+            return <InlineFieldRow key={i}>
+              <InlineField
+                label={`Custom Setting (${i})`}
+                aria-label={`Custom Setting (${i})`}
+              >
```

Comment:
We should have a label for `Setting` and `Value`. 
https://developers.grafana.com/ui/latest/index.html?path=/story/forms-inlinefield--multiple